src/app/core/services/datacenter.ts

```diff
@@ -30,6 +30,8 @@ export class DatacenterService {
   private _seedSettings$ = new Map<string, Observable<SeedSettings>>();
   private _seedSettingsRefresh$ = new Subject<void>();
   private _seedsRefresh$ = new Subject<void>();
+  private _adminSeeds$: Observable<any[]>;
```

Comment:
`any` type?

Reply:
Ah, good catch. Forgot to change during development, will adjust in all places